mk/docs.mk

```diff
@@ -265,3 +269,11 @@ endif
 
 docs: $(DOC_TARGETS)
 compiler-docs: $(COMPILER_DOC_TARGETS)
+
+RUSTC = $(STAGE2_T_$(CFG_BUILD)_H_$(CFG_BUILD))
+
+trpl: $(RUSTBOOK_EXE) $(wildcard $(S)/src/doc/trpl/*.md)
```

Comment:
okay, so i pushed a commit that does the tmp/trpl.ok stuff, but I accidentally forgot to `touch mk/docs.mk`, and it's late, so I'm not gonna do an actual re-build right now. 

what's the best way to include it into `make`, do you think? And, I need to make the mirror `test` rule too.